Sources/MapboxNavigation/CameraController.swift

```diff
@@ -101,7 +101,9 @@ class CameraController: NavigationComponent, NavigationComponentDelegate {
         case .transitionToFollowing, .following:
             navigationViewData.navigationView.overviewButton.isHidden = false
             navigationViewData.navigationView.resumeButton.isHidden = true
-            navigationViewData.navigationView.wayNameView.isHidden = false
+            if let roadName = navigationViewData.navigationView.wayNameView.text {
```

Comment:
This check should also account for an empty string. MapboxNavigation has a handy shortcut for getting a non-empty string, to simplify the logic a bit. Also, `roadName` appears to be unused:

```suggestion
            if let _ = navigationViewData.navigationView.wayNameView.text?.nonEmptyString {
```

Reply:
I guess the `isEmpty` check below makes it unlikely that the name would be an empty string here, but being consistent here gives us more confidence that a future refactor won’t reintroduce the blank bubble.

Reply:
Thanks for the suggestion, I'll improve this one.